docs/user-guide/configure-tls.md

```diff
@@ -108,50 +107,54 @@ be specified as separate backends.
 
 ### Backend Application Attributes:
 
-  - trustAllCerts - Whether (true) or not (false) to authenticate server side.
+
+  - trustAllCerts - Turns off (true) or on (true) client-side TLS authentication. When `trustAllCerts` is false,
```

Comment:
“Client-side TLS authentication” is potentially confusing. It can lead readers to think about TLS client authentication which is a completely different matter. Please paraphrase and remove “client side” etc from the description.